src/test/java/org/sert2521/gamename/Drivetrain/DriveStraightTest.kt

```diff
@@ -0,0 +1,84 @@
+package org.sert2521.gamename.Drivetrain
+
+import org.fest.assertions.Assertions.assertThat
+import org.fest.assertions.Delta
+import org.junit.Before
+import org.junit.Test
+import org.sert2521.gamename.Drivetrain.commands.DriveStraight
+import org.strongback.command.CommandTester
+import org.strongback.components.Motor
+import org.strongback.components.Motor.compose
+import org.strongback.drive.TankDrive
+import org.strongback.mock.Mock
+
+/**
+ * Created by AMD on 6/17/17.
+ */
+class DriveStraightTest() {
+    private val TOLERANCE = Delta.delta(0.001)
+    private val START_TIME_MS: Long = 1000
+
+    private var frontLeft: Motor = Mock.stoppedMotor()
+    private var frontRight: Motor = Mock.stoppedMotor()
+    private var rearLeft: Motor = Mock.stoppedMotor()
+    private var rearRight: Motor = Mock.stoppedMotor()
+
+    private var left: Motor = compose(frontLeft, rearLeft)
+    private var right: Motor = compose(frontRight, rearRight)
+
+    private var drive: TankDrive = TankDrive(left, right)
```

Comment:
How do I not initialize them? If I do this:

```
private var frontLeft: Motor
    private var frontRight: Motor
    private var rearLeft: Motor
    private var rearRight: Motor

    private var left: Motor
    private var right: Motor

    private var drive: TankDrive
```

IntelliJ gets mad: "Property must be initialized or be abstract"